tutorials/2.5/graphics-vertex-array.php

```diff
@@ -77,7 +77,7 @@
 // no texture coordinates here, we'll see that later
 </code></pre>
 <p>
-    Your triangle is ready and you can now draw it. Drawing a vertex array can be done similar to drawing any other SFML entity, by using the <code>draw</code> function:
+    Your triangle is ready and you can now draw it. Drawing a vertex array can be done similarly to drawing any other SFML entity, by using the <code>draw</code> function:
```

Comment:
I think this was actually correct. It could be "similarly to cars" but since it's a verb "similar to eating" is correct, then again, I'm not a native speaker.

Reply:
I would say that similarly is for verbs and similar is for nouns, to be honest.
"vans are similar to cars"
"drawing similarly"

Reply:
However, if you want me to just remove this modification from the PR, let me know.

Reply:
Here, "similarly" refers to the construction "can be done" and none of the two "drawing".
* _It can be done similarly_
* _It is done similarly_
* _One does it similarly_

"similarly" should be correct. [This SO thread](https://english.stackexchange.com/q/189825) discusses a similar topic 😉